Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ endif ()
 #
 if (DEVELOPER_MODE AND (SKIA_ROOT_DIR OR NOT "$ENV{SKIA_ROOT_DIR}" STREQUAL ""))
     if (NOT SKIA_ROOT_DIR)
-        set(SKIA_ROOT_DIR "$ENV{SKIA_ROOT_DIR")
+        set(SKIA_ROOT_DIR "$ENV{SKIA_ROOT_DIR}")
```

Comment:
Instead of this, let's remove the support for building WebKit using an externally built copy of Skia. We haven't been using that for months and we would rather not spend extra effort supporting a different way of building&mdash;the situation with Skia has been stable, and the internal build has worked for us.

Reply:
Sounds reasonable. I can give it a try. Can you create a corresponding bug report for this change that I could refer to in this PR, perhaps?

Reply:
@valgur Here goes: https://bugs.webkit.org/show_bug.cgi?id=284871